tslint.json

```diff
@@ -15,7 +15,8 @@
         ],
         "max-classes-per-file": false,
         "object-literal-sort-keys": false,
-        "no-console": false
+        "no-console": false,
+        "max-line-length": [true, 140]
```

Comment:
Monitors are wide, but lots of people like to code with two panes.

We should settle on what to do about tslint. People's tolerance for it varies widely across the team.